### ISSUE TITLE
Error when running ceci on a page with pre-rendered ceci-elements

### ISSUE THREAD

**Issue by secretrobotron** [mozilla/ceci#45]
e.g. http://overjoyed-crow-216.appalot.me/index.html

During subscription setup:

```
Uncaught TypeError: Cannot set property 'listeningFunction' of undefined ceci.js:289
```

Could be caused by just copying html from rendered ceci element, assuming it will work as new in another page.

**Comment by Pomax** [comment]
this is caused by using the already converted HTML from the designer. Some new functionality has been introduced that lets you dehydrate/rehydrate entire cards -- see the /testapp for an example of how to do this, https://github.com/mozilla/appmaker/blob/master/views/testapp.ejs

**Comment by secretrobotron** [comment]
+1. Fixed in mozilla/appmaker@e8a47aed7484ec288e2e9794001294f672d4f8e2
